setup.cfg

```diff
@@ -19,7 +19,7 @@ exclude = setup.py, setup-raw.py docs/source/conf.py
 disable = duplicate-code,too-many-arguments,too-many-public-methods,too-few-public-methods,
     broad-except,too-many-instance-attributes,fixme,missing-function-docstring,
     missing-class-docstring,too-many-locals,too-many-lines,too-many-branches,
-    too-many-statements
+    too-many-statements, cyclic-import
```

Comment:
Is this necessary only for the examples or for something else as well?

Reply:
************* Module telegram.warnings
telegram/warnings.py:1:0: R0401: Cyclic import (telegram.ext -> telegram.ext._dictpersistence) (cyclic-import)
telegram/warnings.py:1:0: R0401: Cyclic import (telegram.ext._application -> telegram.ext._conversationhandler) (cyclic-import)
telegram/warnings.py:1:0: R0401: Cyclic import (telegram.ext -> telegram.ext._picklepersistence) (cyclic-import)

it's for this, which makes no sense, and adding ignore in that file didn't work.

Reply:
ah okay. I vaguely recall that we had similar problems sometime before … thanks for the explanation! :)